ui/narrative/methods/export_json_to_staging/spec.json

```diff
@@ -0,0 +1,76 @@
+{
+  "ver" : "",
+  "authors" : ["gaprice"],
+  "contact" : "https://www.kbase.us/support/",
+  "visible" : true,
+  "categories" : ["active","util"],
+  "widgets": {
+    "input": null,
+    "output" : "no-display"
+  },
+  "parameters" : [
+    {
+      "id" : "input_ref",
+      "optional" : false,
+      "advanced" : false,
+      "allow_multiple" : false,
+      "default_values" : [ "" ],
+      "field_type" : "text",
+      "text_options" : {
+        "valid_ws_types" : ["*"]
```

Comment:
Noting here that this won't do much and look like an error until https://kbase-jira.atlassian.net/browse/UIP-48 is done.

Reply:
yep, understood